### DISCUSSION TITLE
segmentation fault (core dumped) error

### DISCUSSION THREAD

**Discussion by karkir0003** [python-poetry/poetry#8494] [Q&A]
I'm trying to run a github action to install poetry dependencies and cache the venv within mamba. Here is a [link](https://github.com/DSGT-DLP/Deep-Learning-Playground/blob/nextjs/.github/workflows/backend.yml) to the github actions yml file and a [link](https://github.com/DSGT-DLP/Deep-Learning-Playground/actions/runs/6371725006/job/17293916310?pr=965) to the failed run. Could someone take a look and provide me with pointers on how to debug the cause of the segfault

**Comment by deronnax** [comment]
I would say first, pass a `-vvv` flag to `poetry install` to know what went wrong.

**Reply by karkir0003** [reply]
Hi @deronnax. Thanks for responding. I was able to resolve this issue by using a different github action to structure and cache the mamba env. I used this [action](https://github.com/marketplace/actions/setup-miniconda)

**Reply by karkir0003** [reply]
I previously used `setup-micromamba` github action and the docs were unclear there, so I shifted to the action I linked above
